doc/tutorials/tutorial_X8.rst

```diff
@@ -261,16 +273,16 @@ Furthermore, several automatic decompositions are supported:
   This performs a rectangular decomposition using Mach-Zehnder interferometers.
 
 * You can use :class:`~.ops.BipartiteGraphEmbed` to embed a bipartite graph on
-  the photonic chip.
+  the photonic device.
 
   .. warning::
 
       Decomposed squeezing values depend on the graph
       structure, so only bipartite graphs that result in equal squeezing on all
-      modes can be executed on currently available chips.
+      modes can be executed on ``X8_01`` chips.
```

Comment:
```suggestion
      modes can be executed on ``X8`` chips.
```

Reply:
I feel like we're not clear on devices vs chips and X8 vs X8_01.
Is X8 a family of 8-mode chips (maybe this is akin to a "device") and X8_01 a particular chip?

Reply:
Yep, exactly! I will add a paragraph explaining this.